usage_report/usage_report.py

```diff
@@ -130,13 +130,16 @@ def main(date, lag_days, sample, no_output, input_bucket, input_prefix, input_ve
     # all counts will be multipled by sample_factor
     sample_factor = 100.0 / sample
 
+    # A list of the allowed normalized channels
+    use_channels = ['release', 'esr', 'beta', 'Other']
```

Comment:
This should instead be a constant at the top of the file, and caps: `ALLOWED_CHANNELS`. A good rule of thumb when writing software is "No Magic Numbers" - meaning strings, numbers, and constants should all be defined for a reason. The name of the constant should give the reason.

This is why you'll see scientific software with constants like `SECONDS_PER_MINUTE`, to give an idea where the magic string came from (60 just floating around in code would get very confusing, and wouldn't be clear what it's referencing).

Finally, it makes it easier to update. With constants defined at the top of the file there's just simple code changes with no hunting when making a configuration change, e.g. adding a new channel.

Reply:
Got it, made the change.